gt7-extramenus.py

```diff
@@ -102,7 +102,7 @@ def start_exploit(section):
     press("escape")
     press("escape")
     press("escape")
-    sleep(4)
+    sleep(2.5)
```

Comment:
Please discard this change, in my testing, sometimes it could take a bit longer to load the menu.